packages/web-components/src/index.html

```diff
@@ -18,6 +18,43 @@
 
   </head>
   <body>
-    <my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>
```

Comment:
I apologize for not mentioning this earlier, but could we revert the changes to this file? 😅

Reply:
oops! sorry! Done